pywbem_mock/_wbemconnection_mock.py

```diff
@@ -480,6 +481,73 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         self._merge_repos(mof_repo)
 
+    def compile_dmtf_schema(self, dmtf_schema_version, schema_dir, classes,
+                            use_experimental=False, verbose=False):
+        """
+        Compile the classes defined by `classes` and their dependent classes
+        from the DMTF CIM schema version defined by 'dmtf_version_version' and
+        keep the downloaded DMTF CIM schema in the directory defined by
+        `schema_dir`.
+
+        This method uses :class:`~pywbem_mock.DMTFCIMSchema` to get the schema
+        from the DMTF, save it in the `schema_dir` directory, expand it and make
+        a list of leaf classes from this directory that are to be compiled.
+
+        It automatically compiles the DMTF qualifier declarations.
+
+        Parameters:
+
+          schema_version  (tuple of 3 integers (m, n, n):
```

Comment:
The parameter name used in the docstring does not match the declared parameter name.

Reply:
Found that one. Fixed